src/lib/y2network/interface_type.rb

```diff
@@ -134,5 +136,19 @@ def method_missing(method_name, *arguments, &block)
     HSI = new(N_("HSI"), "hsi")
     # FICON-attached direct access storage devices. s390 specific
     FICON = new(N_("FICON"), "ficon")
+    # Point-to-Point Protocol
+    PPP = new(N_("Point-to-Point Protocol"), "ppp")
+    # Ip in Ip protocol
+    IPIP = new(N_("Ip-in-ip"), "ipip")
+    # IPv6 Tunnel interface
+    IPV6TNL = new(N_("IPv6 Tunnel"), "ip6tnl")
+    # IPv6 over IPv4
+    SIT = new(N_("IPv6 over IPv4 Tunnel"), "sit")
+    # IP over IPv4
+    GRE = new(N_("IP over IPv4 Tunnel"), "gre")
+    # Infrared
+    IRDA = new(N_("Infrared"), "irda")
+    # Loopback
+    LO = new(N_("Loopback"), "lo")
```

Comment:
wow, that is a lot of new types, can we somehow handle it? we need to add at least popup that we cannot configure it.

Reply:
none of these types was never created by y2-network. But we were able to detect them (and e.g. filter such ifcfg files out of parsing). However it would be nice to have ability to e.g. create these IP* over IP* tunnel devices in the future.